[DOC-6100] Document that private clusters are required for CMEK (#15568)
cockroachdb/docs: cockroachcloud/egress-perimeter-controls.md

Regardless of user-specific Egress Perimeter Control policy, egress is always permitted to services that are managed by Cockroach Labs and are essential to your cluster's functionality and ongoing operations.
{{site.data.alerts.end}}

> ## Prerequisites

Author: FYI @MichaelTrestman we are changing the below wording in both the EPC and CMEK topics. I've also simplified the Prerequisites section heading in EPT.

- You need a {{ site.data.products.dedicated }} cluster. Egress Perimeter Controls are not supported for {{ site.data.products.serverless }} clusters.
- Your cluster must be a **Private Cluster**, with no public IP addresses on its nodes. Refer to [Private Clusters](private-clusters.html).

> - You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts and API keys in the cloud console. Refer to [Service Accounts](console-access-management.html#service-accounts).

Reviewer: nit: since GCP's admin pane is called the "Google Cloud Console" (as used in `use-cloud-storage-for-bulk-operations.md`, for example), we should clarify which cloud console we mean here: likely "CockroachDB Cloud Console" (as used in `vault-db-secrets-tutorial.md` and `connect-to-the-database.md` for example).

> - You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts and API keys in the cloud console. Refer to [Service Accounts](console-access-management.html#service-accounts).

Author: ```suggestion
- You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts and API keys in CockroachDB Cloud Console. Refer to [Service Accounts](console-access-management.html#service-accounts).
```

- You need a {{ site.data.products.db }} account with billing enabled. Egress Perimeter Controls are available only for {{ site.data.products.dedicated }}, not {{ site.data.products.serverless }} clusters, and {{ site.data.products.dedicated }} have a non-zero operating cost that must be billed.

- Your cluster must be a **Private Cluster**, with no public IP addresses on its nodes. Refer to [Private Clusters](private-clusters.html).

> - You need an API key with `admin` privilege on clusters in your organization. You can provision API keys in the cloud console: [Create API Keys](console-access-management.html#create-api-keys)

Reviewer 2: ```suggestion
- You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts and API keys in the cloud console: [Service Accounts](console-access-management.html#service-accounts)
```
